src/openfl/display/Bitmap.hx

```diff
@@ -384,7 +384,7 @@ class Bitmap extends DisplayObject implements IShaderDrawable {
 		
 		if (__bitmapData != null) {
 			
-			if (value != __bitmapData.height) {
+			if (value != __bitmapData.height * __scaleY) {
```

Comment:
Note that it intentionally doesn't do `Math.abs` on scaleX/Y values there, because that's how Flash behaves:

```as3
trace(bitmap.bitmapData.width); // 250
bitmap.scaleX = -1;
trace(bitmap.width); // 250
bitmap.width = bitmapData.width;
trace(bitmap.width); // 250
trace(bitmap.scaleX); // 1
```